en/log/flight_review.md

```diff
@@ -180,6 +180,30 @@ Very high (unsafe) vibration levels.
 ![Exceedingly high vibration in raw accel. plot](../../assets/flight_log_analysis/flight_review/vibrations_exceedingly_high_accel.png)
 
 
+### Raw hHgh-rate IMU Data Plots {#fifo_logging}
+
+For an in-depth analysis there is an option to log the raw IMU data at full rate (several kHz, depending on the IMU).
+This allows inspection of much higher frequencies than with normal logging, which can help when selecting vibration mounts or configuring low-pass and notch filters appropriately.
+
+To use it, some parameters need to be changed:
+- Set [IMU_GYRO_RATEMAX](../advanced_config/parameter_reference.md#IMU_GYRO_RATEMAX) to 400.
+  This is required to reduce the amount of logged data
+- Use a good SD card, as the IMU data requires a high logging bandwidth
+  > **Tip** See [Logging > SD Cards](https://dev.px4.io/master/en/log/logging.html#sd-cards) for a comparison of popular SD card.
+- Enable either the gyro or accel high-rate FIFO profile in [SDLOG_PROFILE](../advanced_config/parameter_reference.md#SDLOG_PROFILE) and disable the rest of the entries.
+  If you are using a really good SD card (Flight Review will show dropouts if the logging rate gets too high), you can:
```

Comment:
Should this bit "(Flight Review will show dropouts if the logging rate gets too high)," move up into my note on line 192 - i.e. the tip is "Here are some good SD cards, but you know they are not good enough when you start seeing dropouts?

Reply:
Yes we can move it up, it applies in general, but particularly if you want to max it out.

Reply:
OK, I moved it up. This is the first place we mention using a good SD card, so to me this is the right place to mention how you know that it is "not good enough).